RELEASE.md

```diff
@@ -1,23 +1,233 @@
+   checkout branch-0.13.x
+   commit id: "Backport fix C"
+   commit id: "Release 0.13.1" tag: "v0.13.1"
+   commit id: "Backport fix D"
+   checkout main
+   commit id: "Feature B"
+   commit id: "Fix C"
+   commit id: "Release 0.14" tag: "v0.14.0"
+   branch branch-0.14.x
+   checkout branch-0.14.x
+   commit id: "Backport fix D " %% Space at the end because mermaid doesn't 
+   %% allow duplicate labels and only displays one of the commits with duplicate labels...
+   commit id: "Release 0.14.1" tag: "v0.14.1"
+   checkout main
+   commit id: "Fix D"
+```
+
+## Version bump commits
+
+For simplicity the graph in previous section suggested that version bumps happen in one commit.
+In reality we bump each crate version in a separate commit.
+
+For an example, see four last commits of this PR: https://github.com/scylladb/scylla-rust-driver/pull/1040/commits
+
+Commits that update `scylla-proxy` / `scylla-macros` / `scylla-cql` change:
+- `version` field in relevant `Cargo.toml`
+- Version number in crates that depend on given crate (so a commit bumping `scylla-macros` will also change version of 
+`scylla-macros` in `scylla` and `scylla-cql` crates).
+- Version number of the crate in `Cargo.lock.msrv`
+
+Additionally the last commit (which bumps `scylla` crate version) changes:
+- `docs/pyproject.toml` and `docs/source/conf.py`, as described in "Documentation" section.
+- Version number in `docs/source/quickstart/create-project.md`.
+
+
+## Backporting
+
+If you want to backport a whole PR, you should chery-pick the merge commit. You will need `-m` flag for that - read about it in the manual.
+
+You can also backport just a specific commit / commits of bigger PR, also using cherry-pick.
+
+Please use `-x` flag for cherry-pick to record information about original commit.
+
+## Releasing
+
+IMPORTANT: Read this whole document before attempting to do a release.
+
+Prerequisites:
+- Have a write access to GitHub repo (so that you are able to push to main and create a new branch).
+- Have access to our crates on [crates.io](https://crates.io).
+- For each of our 3 crates in this workspace (`scylla-macros`, `scylla-cql`, `scylla`) decide if it needs a new version, and if so, what is the correct version number. See "Crates and versions" section.
+
+### Releasing a major version
+
+1. Check out current `main`, e.g. `git fetch; git checkout origin/main`.
+2. Create a new version branch, e.g. `git checkout -b "branch-0.14.x"`.
+3. Create commits that bump version numbers of the crates (see `Version bump commits` section in this document).
+4. Push the new branch to our repo (e.g. `git push origin branch-0.14.x`).
+5. Prepare release notes (see the template at the bottom of this document).
+6. (Optional, but recommended) Create a PR from the new branch to `main` so that another maintainer can review your work.
+Description of this PR should consist of just release notes. Don't merge this PR using GitHub interface - just wait for approval.
```

Comment:
Let's make this warning stand out:

Reply:
`**NOTE: Don't merge this PR using GitHub interface - just wait for approval.**`

Reply:
...and optionally provide a short explanation why merging using the GH UI is wrong.

Reply:
I'll add this.

I'm just unable to understand how it is possible that GitHub doesn't support fast-forward merges in UI. It's absurd.